src/Datapackages/BaseDatapackage.php

```diff
@@ -210,19 +217,24 @@ public function save($zip_filename)
         }
     }
 
+    protected function copy()
```

Comment:
Since you are here, this could need a PHP DOC saying what it does. I know it is obvious but makes future contributor's work easier. Like the fact that it creates a new Object but skips validation.

Reply:
Right, docblock, of course. I can add that. I can fix that in a new commit and put it in this same pull request, right?

Reply:
Yes! Once the pull is open you can keep pushing commits into your local/forked same branch.